lib/install.js

```diff
@@ -135,9 +136,15 @@ function symlinkSelf (target, name, depth) {
  *     symlinkToModules(fullname, name, modules, 0)
  */
 
-function symlinkToModules (target, name, modules) {
+function symlinkToModules (target, pkg, modules) {
   // lodash -> .store/lodash@4.0.0
   // .store/foo@1.0.0/node_modules/lodash -> ../../../.store/lodash@4.0.0
   // .tmp/01234567890/node_modules/lodash -> ../../../.store/lodash@4.0.0
-  return relSymlink(target, join(modules, name))
+  if (pkg.scope) {
+    debug('make scope dir', pkg.scope)
+    return fs.mkdir(join(modules, pkg.scope))
```

Comment:
oops, there's a bug here, you now can't install more than 1 module from the same scope.

```
pnpm install @rstacruz/tap-spec@* @rstacruz/deku@*
```